contrib/build-wine/deterministic.spec

```diff
@@ -36,6 +36,7 @@ datas = [
     (f"{PROJECT_ROOT}/{PYPKG}/locale", f"{PYPKG}/locale"),
     (f"{PROJECT_ROOT}/{PYPKG}/plugins", f"{PYPKG}/plugins"),
     (f"{PROJECT_ROOT}/{PYPKG}/gui/icons", f"{PYPKG}/gui/icons"),
+    (f"{PROJECT_ROOT}/{PYPKG}/gui/fonts", f"{PYPKG}/gui/fonts"),
```

Comment:
The mac build also uses pyinstaller, so probably needs an analogous line here: https://github.com/spesmilo/electrum/blob/77a00db52a752226540fed0d9cafffe5a2525a95/contrib/osx/osx.spec#L41

Reply:
I've built a macos binary. The fonts are included as expected.
```
% ls -lah dist/Electrum.app/Contents/Resources/electrum/gui/fonts 
total 656
drwxr-xr-x  5 vagrant  staff   160B  3 Mar 17:03 .
drwxr-xr-x  4 vagrant  staff   128B  3 Mar 17:03 ..
-rw-r--r--  1 vagrant  staff   149K 22 Jan  2001 PTMono-Bold.ttf
-rw-r--r--  1 vagrant  staff   166K 22 Jan  2001 PTMono-Regular.ttf
-rw-r--r--  1 vagrant  staff   4.3K 22 Jan  2001 PTMono.LICENSE
```